e2e/_suites/fleet/README.md

```diff
@@ -141,6 +141,30 @@ Background: Setting up kibana instance with default profile
 ```
 3. Run the tests! Kibana will be recreated with the profile configuration in those scenarios using the new step.
 
+### Fleet UI e2e tests CI job
+
+https://beats-ci.elastic.co/job/e2e-tests/job/e2e-testing-kibana-fleet/build?delay=0sec
+
+### Running against a Kibana pull request locally
+
+1. Build kibana docker image from pull request using this job: (Custom Kibana - Deploy)[https://apm-ci.elastic.co/job/apm-shared/job/oblt-test-env/job/custom-kibana-deploy/build?delay=0sec]
+   - Provide `kibana_branch` parameter to refer to your pr number e.g. `PR/100000`
+   - Skip deploy_kibana step
+2. Set envvar to pr
+`export KIBANA_VERSION=pr100000`
+3. Run tests
+
+### Running against a Kibana running locally
+
+1. Set envvars
+```
+export PROVIDER=remote
+export KIBANA_URL=http://localhost:5601
+export ELASTICSEARCH_URL=http://localhost:9200
+export FLEET_URL=http://localhost:8220
```

Comment:
@adam-stokes is this needed? IIRC the Docker provider is still enabled for local development, so no need to run against a remote kibana locally. Please correct me if I'm wrong

Reply:
Yea this is needed now, I've got a branch im working on that makes all this seemless